### PR TITLE
Adding a partner api for user creation (issue 146)

### LINKED ISSUE #146: API para parceiros enviar pré assinaturas

### PR DESCRIPTION
**Pull request by erickzanardo**
This PR closes #146

### How was it before?

- the partners did not had any means to send their users to mudamos

### What has changed?

- an api was added for the partners to create users on mudamos-web platform

### What should I pay attention when reviewing this PR?

Nothing special

### Is this PR dangerous?

no

### CODE REVIEW

**Review by marcoamarelo — COMMENTED** [review]
Erick, foi simples assim essa funcionalidade? Tem um exemplo de chamada pra eu ver como ficou?

File: app/controllers/partners_api/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  def create
+    user_params = params.permit(:email, :name, :encrypted_password)
+    user = user_service.create_user_with_password(
+      email: user_params["email"],
+      name: user_params["name"],
+      encrypted_password: user_params["encrypted_password"]
+    )
+    
+    if user.valid?
+      plips = plip_repository.all_initiated(page: 1, limit: 1).items
```
Comment (guimello) [inline app/controllers/partners_api/users_controller.rb]: Or you could add this method on the repository `#current_plip`.

**Review by guimello — APPROVED** [review]
🚀

File: app/controllers/partners_api/users_controller.rb (hunk 1 above)
Comment (erickzanardo) [reply app/controllers/partners_api/users_controller.rb]: good ideia

**Comment by erickzanardo** [comment]
@marcoamarelo a api pode ser chamada assim:

`POST http://localhost:3000/partners_api/users`

Headers
`Authorization: Bearer TOKEN_OBTIDO_POR_OAUTH`

Payload:
```
{
  "email": "yoshi@yoshi.com",
  "name": "Yoshi",
  "encrypted_password": "$2a$10$tKMS.By4l4GPjbkVt4cewO58939Iy4Gq68Cq3PN87ZnO8nkpsIJdy"
}
```

**Pull request merged**